src/pages/_app.js

```diff
@@ -2,12 +2,14 @@ import Layout from "@/components/shared/layout/layout";
 import "@/styles/globals.css";
 import { config } from "@fortawesome/fontawesome-svg-core";
 import "@fortawesome/fontawesome-svg-core/styles.css";
+import { GoogleAnalytics } from "nextjs-google-analytics";
```

Comment:
cool! ive never used this package. looks like it makes it really easy. 

the docs have instructions on how to remove tracking from non production environments:
```
Your Google Analytics measurement id is read from NEXT_PUBLIC_GA_MEASUREMENT_ID environment variable, so make sure it is set in your production environment:

If the variable is not set or is empty, nothing will be loaded, making it safe to work in development.

To load it and test it on development, add:
NEXT_PUBLIC_GA_MEASUREMENT_ID="G-XXXXXXXXXX"
to your .env.local file.
```